### ISSUE TITLE
Create advanced feature importance ranking as guide for implementers

### ISSUE THREAD

**Issue by goeddea** [wamp-proto/wamp-proto#102]
There are features in the Advanced Spec which are of greater practical importance than others.

As an example:

The combination of [caller identification](https://github.com/tavendo/WAMP/blob/master/spec/advanced.md#caller-identification) and [subscriber exclusion](https://github.com/tavendo/WAMP/blob/master/spec/advanced.md#subscriber-black--and-whitelisting) allows to exclude a caller from receiving a publication which contains an update  based on the call. This is a recurring pattern, which allows a component to treat changes triggered internally and externally differently. There is no easy way to otherwise achieve the same result.

We want to indicate to implementers which features are important, and why. 

This can e.g. be done by adding a list of features, grouped into importance categories, and giving one or more examples each for where they are needed in practice.

**Comment by estan** [comment]
Yes. I think the WAMP spec is pretty big now. And especially the Advanced part if you include everything non-`[stable]` is pretty daunting to look at for implementers (where to start?). So I think this is a good idea.
